tests/perf-test/SelectionList.perf-test.tsx

```diff
@@ -88,8 +97,7 @@ function SelectionListWrapper(args) {
             onSelectRow={onSelectRow}
             initiallyFocusedOptionKey="item-0"
             ListItem={RadioListItem}
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {...args}
+            canSelectMultiple={canSelectMultiple}
```

Comment:
Why this change?

Reply:
I've noticed that `canSelectMultiple` is the only prop provided to `SelectionListWrapper`, so I decided to simplify it